ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml

```diff
@@ -38,7 +38,7 @@ tests:
     env:
       CHANNEL_GROUP: candidate
       CHURN: "false"
-      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      COMPUTE_MACHINE_TYPE: m5.xlarge
```

Comment:
@liqcui If you do this change, I think openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-candidate-x86-loaded-upgrade-from-4.12.yaml should be changed to make them consistent. In rosa-aws-sts workflow the env is set to COMPUTE_MACHINE_TYPE: "m5.2xlarge"

Reply:
Fixed in new code, thanks!

Reply:
@liqcui lgtm now. Let's see if @svetsa-rh and @paigerube14 have other comments.

Reply:
I think theres' a larger effort to align different tests to have similar compute/master node sizes. I dont think this pr is where we should be updating that. Can we stick to only adding the APF tests and put this into discussion in another pr?

Reply:
Hi Paige, I can move the change to my another rosa loaded upgrade pr

Reply:
@paigerube14 I have move the change to another rosa loaded upgrade pr, please review again, thanks!